src/test/wrappers/test_cobs.py

```diff
@@ -36,6 +38,25 @@ def test_creating_new_signature_sizes(make_classic_index_dir):
     assert len(list(tmp_classic_index_dir.glob('*'))) == 15
 
 
+@integration_test
+def test_renaming_samples(make_classic_index_dir):
```

Comment:
Can we add a test to make sure that if we did not update sample names in at least one of the index files, these index files will still be the same, e.g. having the same md5 sum as before?